stories/002.CreatingAComponent.stories.mdx

```diff
@@ -65,11 +65,11 @@ This document describes the creation of a basic Nightingale component.
 
 5. Create the entrypoint for webpack. The current setup of the nightingale project looks for `src/index.ts`:
 
-  ```typescript
-  import NightingaleHello from "./nightingale-hello";
+```typescript
+import NightingaleHello from "./nightingale-hello";
```

Comment:
Block is no longer aligned with other blocks on the same level

```suggestion
    ```typescript
    import NightingaleHello from "./nightingale-hello";
```